addOns/savehar/src/main/java/org/zaproxy/zap/extension/savehar/PopupMenuItemSaveHarMessage.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2020 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.saveharmessage;
+
+import edu.umass.cs.benchlab.har.HarEntries;
+import edu.umass.cs.benchlab.har.HarLog;
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.text.MessageFormat;
+import javax.swing.JFileChooser;
+import javax.swing.filechooser.FileFilter;
+import org.apache.log4j.Logger;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.model.Model;
+import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.view.View;
+import org.zaproxy.zap.utils.HarUtils;
+import org.zaproxy.zap.view.popup.PopupMenuItemHttpMessageContainer;
+import org.zaproxy.zap.view.widgets.WritableFileChooser;
+
+public class PopupMenuItemSaveHarMessage extends PopupMenuItemHttpMessageContainer {
+
+    private static final long serialVersionUID = -7217818541206464572L;
+
+    private static final Logger log = Logger.getLogger(PopupMenuItemSaveHarMessage.class);
```

Comment:
```suggestion
    private static final Logger LOG = Logger.getLogger(PopupMenuItemSaveHarMessage.class);
```

This is also a constant.